src/core/client/stream/tabs/Comments/Stream/AddACommentButton.tsx

```diff
@@ -26,6 +37,30 @@ const AddACommentButton: FunctionComponent<Props> = ({
   isQA = false,
   currentScrollRef,
 }) => {
+  const pollForCommentFormElement = useCallback(
+    async () =>
+      new Promise<HTMLElement>((resolve, reject) => {
+        let count = 0;
+        const commentFormPolling = setInterval(async () => {
+          count += 1;
+          const postCommentFormElement =
+            root.getElementById(POST_COMMENT_FORM_ID);
+          if (
+            postCommentFormElement !== undefined &&
+            postCommentFormElement !== null
+          ) {
+            clearInterval(commentFormPolling);
+            resolve(postCommentFormElement);
+          }
+          if (count > 10) {
+            clearInterval(commentFormPolling);
+            reject("Unable to locate comment form");
+          }
+        }, 100);
+      }),
+    []
+  );
```

Comment:
Not bashing you for having to do this as Kathryn has had to do similar in the past to make keyboard shortcuts work... but it makes me sad that React has made us come to this. 😢